routes/dockerflow.js

```diff
@@ -15,7 +15,7 @@ try {
   console.error("Error in git-rev-sync: ", err);
 }
 
-const versionJsonPath = path.join(__dirname, "..", "public", "version.json");
+const versionJsonPath = path.join(__dirname, "..", "version.json");
```

Comment:
I'm getting `EACCES: permission denied` here 😢 ... I can't tell why.

```
$ docker run -p 6060:6060 blurts-server
fs.js:646
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: EACCES: permission denied, open '/app/version.json'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.writeFileSync (fs.js:1299:33)
    at Object.<anonymous> (/app/routes/dockerflow.js:25:4)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
```

Reply:
Only when running this via `docker run -p 6060:6060 blurts-server` (or `npm run docker:run`)

Reply:
Fixed this by adding `version.json` to the `.dockerignore` file to stop it being copied from the host machine (with host machine perms) to the container.